### DISCUSSION TITLE
Help for filling the education application

### DISCUSSION THREAD

**Discussion by Quartz-niloy** [community/community#137856] [GitHub Education]
### Hi everyone,

Hi, so uh the photo needs to have both the information and a date of enrollment but in my university ID, we don't have an enrollment date rather an expiration one that too on the backside of the ID card but the application takes only one photo. Could a Staff or anyone kindly help or suggest me anything in this regard. I have the admission slip but that's in bangla. Any help will be appreciated

**Comment by davevad93** [comment]
Hi @Quartz-niloy ,

To avoid rejection, some sort of academic documentation with a current date in `DD/MM/YYYY` or `MM/DD/YYYY` format is required. As stated in the [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student), documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter. If your student ID doesn't have a current date, you can provide some of the above documents.

**Reply by Quartz-niloy** [reply]
I get that, but that's on the backside of my uni ID where there's no picture of me or anything like that and while taking the picture you can't take 2, what can I do for that? SHould I use other forms of identification? All of my other identifications uses Bangla not English.

**Reply by davevad93** [reply]
Hey @Quartz-niloy yes, you should try one of the academic documents listed above, but you have to translate them to English otherwise they will be rejected.

**Comment by joemar25** [comment]
Hi @Quartz-niloy , I understand your concern. Since your university ID doesn't have an enrollment date on the front side, and the application only allows one photo,you can consider providing alternative academic documents that prove your current student status.

As mentioned earlier, these documents can include a class schedule, transcript, or an enrollment verification letter. If you have any of these documents, you can take a clear photo of them and upload it to the application. However, please note that if your documents are in Bangla, you'll need to translate them to English to ensure they're not rejected. You can try to get an official translation from your university or a certified translation service.

If you're still unsure, you can also reach out to the application support team to ask about their specific requirements and see if they can provide any additional guidance.
